types.go

```diff
@@ -81,6 +81,12 @@ type BeatmapInfo struct {
 	Checksum     string
 }
 
+// This function is used to check whether or not the map is ranked/approved (for clients before b388)
```

Comment:
I would follow the "golang-style" of comments here, so for example:
```suggestion
// IsRanked is used to check whether or not the map is ranked/approved
```